pkg/kube/webhooks/eirini/volume_webhook.go

```diff
@@ -0,0 +1,60 @@
+package eirini
```

Comment:
maybe don't include the eirini dir here: https://github.com/SUSE/eirini-extensions/tree/7a64d9f5f0d20643c3475d641ab48e2d0fa40d6c/pkg/kube/webhooks

and have the package name be `webhooks`

then `eirini.Volume(..)` becomes `webhooks.Volume(..)` which makes a bit more sense I think

wdyt?

Reply:
:+1:  Good idea. Was actually wondering the same, at the end this project will host only eirini-related code